requirements/requirements_nlp.txt

```diff
@@ -8,7 +8,6 @@ unidecode
 youtokentome
 numpy
 tqdm
-sklearn
```

Comment:
and nlp callbacks?

Reply:
since its already in requirements.txt nlp collection should inherit it no?

Reply:
yes installing nemo["nlp"] installs both requirements.txt and requirements_nlp.txt
installing nemo["asr"] installs both requirements.txt and requirements_asr.txt
installing nemo["tts"] installs requirements.txt, requirements_asr.txt, and requirements_tts.txt